dictionaries/dict.json

```diff
@@ -48752,7 +48753,7 @@
 "PAER/SO*PB": "Pearson",
 "PAERBL": "partial",
 "PAERL": "parole",
-"PAERPB": "partner",
```

Comment:
Let's add `"PAERPB": "partner",` to bad habits

Reply:
Done.